src/status_im/ui/screens/accounts/create/styles.cljs

```diff
@@ -42,9 +43,11 @@
   {:size      82
    :icon-size 34})
 
-(def input-container
+(defstyle input-container
   {:margin-horizontal 16
-   :margin-top        105})
+   :margin-top        105
+   :android           {:padding-top 13
```

Comment:
i believe @goranjovic meant `:androuid` map values, so it should be 
```
{:padding-top 13
 :margin-top    92
```

Reply:
Do you mean that the map should begin from the next line?  
I think, however, the current version is consistent with relevant code in the code base.

Reply:
13 and 92 should be aligned

Reply:
Yes, I see the point. I aligned them now. Please let me know if there is any other issue concerning my change. Thanks